src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -840,21 +837,6 @@ private static bool StaticIsFile(UriParser syntax)
         // On by default in .NET 4.5+ and cannot be disabled by config.
         private static volatile bool s_IriParsing = true; // IRI Parsing is always enabled in .NET Native and CoreCLR
```

Comment:
SInce this seems to never be set to false, both it and any path taken when it is false would also be dead code?